docs/troubleshooting/clusterResourcePlacementWorkSynchronized.md

```diff
@@ -0,0 +1,114 @@
+      1 cluster(s)
+    observedGeneration: 1
+    reason: SchedulingPolicyFulfilled
+    status: "True"
+    type: ClusterResourcePlacementScheduled
+  - lastTransitionTime: "2024-05-14T18:05:05Z"
+    message: All 1 cluster(s) start rolling out the latest resource
+    observedGeneration: 1
+    reason: RolloutStarted
+    status: "True"
+    type: ClusterResourcePlacementRolloutStarted
+  - lastTransitionTime: "2024-05-14T18:05:05Z"
+    message: No override rules are configured for the selected resources
+    observedGeneration: 1
+    reason: NoOverrideSpecified
+    status: "True"
+    type: ClusterResourcePlacementOverridden
+  - lastTransitionTime: "2024-05-14T18:05:05Z"
+    message: There are 1 cluster(s) which have not finished creating or updating work(s)
+      yet
+    observedGeneration: 1
+    reason: WorkNotSynchronizedYet
+    status: "False"
+    type: ClusterResourcePlacementWorkSynchronized
+  observedResourceIndex: "0"
+  placementStatuses:
+  - clusterName: kind-cluster-1
+    conditions:
+    - lastTransitionTime: "2024-05-14T18:05:04Z"
+      message: 'Successfully scheduled resources for placement in kind-cluster-1 (affinity
+        score: 0, topology spread score: 0): picked by scheduling policy'
+      observedGeneration: 1
+      reason: Scheduled
+      status: "True"
+      type: Scheduled
+    - lastTransitionTime: "2024-05-14T18:05:05Z"
+      message: Detected the new changes on the resources and started the rollout process
+      observedGeneration: 1
+      reason: RolloutStarted
+      status: "True"
+      type: RolloutStarted
+    - lastTransitionTime: "2024-05-14T18:05:05Z"
+      message: No override rules are configured for the selected resources
+      observedGeneration: 1
+      reason: NoOverrideSpecified
+      status: "True"
+      type: Overridden
+    - lastTransitionTime: "2024-05-14T18:05:05Z"
+      message: 'Failed to sychronize the work to the latest: works.placement.kubernetes-fleet.io
+        "crp1-work" is forbidden: unable to create new content in namespace fleet-member-kind-cluster-1
+        because it is being terminated'
+      observedGeneration: 1
+      reason: SyncWorkFailed
+      status: "False"
+      type: WorkSynchronized
+  selectedResources:
+  - kind: Namespace
+    name: test-ns
+    version: v1
+```
+The `ClusterResourcePlacementWorkSynchronized` condition in the CRP status is flagged as false. It is clear from the message
+that the work object `crp1-work` is prohibited from generating new content within the namespace `fleet-member-kind-cluster-1`
+as it's currently undergoing termination.
+
+### Resolution:
+- To address this specific issue, recreate the Custom Resource Placement (CRP) with a newly selected cluster.
+- Alternatively, delete the CRP and any work on the namespace, then wait for the namespace to regenerate
+
+In other scenarios, you might opt to wait for the work to finish propagating. If the issue persists, consider deleting the CRP and recreating it.

```

Comment:
- We can mention latency as a reason to ask users to wait till work to finish propagating
- Deleting the CRP and re-creating it shouldn't fix any issue by itself in most cases we either wait or ask user to take other actions to resolve right ?